bubuku/features/rebalance/change.py

```diff
@@ -134,11 +135,11 @@ def run(self, current_actions) -> bool:
             return True
         if self.zk.is_rebalancing():
             return True
-        new_broker_ids = sorted(int(id_) for id_ in self.zk.get_broker_ids())
-        if new_broker_ids != self.broker_ids:
-            _LOG.warning("Rebalance stopped because of broker list change from {} to {}".format(
-                self.broker_ids, new_broker_ids))
-            return False
+        # new_broker_ids = sorted(int(id_) for id_ in self.zk.get_broker_ids())
+        # if new_broker_ids != self.broker_ids:
+        #     _LOG.warning("Rebalance stopped because of broker list change from {} to {}".format(
+        #         self.broker_ids, new_broker_ids))
+        #     return False
```

Comment:
No, this should be done in different manner. 
The purpose of this check is to stop rebalance when one of nodes goes down or new node created (For example one decided to restart node). It's ok to remove it when the process is manually started, but with automatic process one should still take care of this situation